docs/querying-data.rst

```diff
@@ -0,0 +1,157 @@
+Querying data in Kingfisher Process
+====================================
+
+The Kingfisher Process :doc:`data-model` is based on collections. Each collection contains either data downloaded from an OCDS publisher using a `Kingfisher Scrape <https://kingfisher-scrape.readthedocs.io/en/latest/>`__ spider, or loaded locally using the :doc:`cli/local-load` cli tool.
```

Comment:
Already covered here: https://kingfisher-process.readthedocs.io/en/latest/#typical-usage

Reply:
Updated in https://github.com/open-contracting/kingfisher-process/pull/220/commits/f147f690aec9099b7e405943b55a780027dc9ee4